CHANGELOG.md

```diff
@@ -53,6 +53,8 @@ database (PR #5260)
 - Fix special character double escaping in instruments (PR #6223)
 
 ### Modules 
+- The dataquery module has a new feature of importing a CSV file (containing either PSCID or DCCID) and for filtering the population by the demographic field.
```

Comment:
This is under the wrong LORIS version. 23 is already released.

And I think we should probably just add a line saying the GUI has been completely redesigned.